inflight/inflight_test.go

```diff
@@ -82,6 +83,25 @@ func TestInflight(t *testing.T) {
 	}
 }
 
+//go:norace
+func TestInflight_AutoShrink(t *testing.T) {
```

Comment:
Usage of the underscore is not conventional in golang. Maybe better to replace it with, for example, `With`?
`TestInflightWithAutoShrink` ?

Reply:
fixed